### PR TITLE
Jk pg search

### PR DESCRIPTION
**Pull request by juliankohlman**
• Steps 
- `bundle install`
- `rails db:reset`
- `rails s`
- route for local => `http://localhost:3001/api/v1/search?query=<your search here>`
🚀

### CODE REVIEW

File: app/controllers/api/v1/businesses_controller.rb [hunk 1]
```diff
@@ -1,70 +1,110 @@
 module Api
-  module V1
-    class BusinessesController < ApplicationController
-      def index
-        @businesses = params[:user_id].present? ? Business.where(user_id: params[:user_id]).first : Business.all
-        render json: @businesses.with_attached_image
-      end
+	module V1
```
Comment (bpleslie) [inline app/controllers/api/v1/businesses_controller.rb]: Might want to configure your IDE to use 2 spaces for indents

File: app/models/business.rb [hunk 2]
```diff
@@ -1,10 +1,24 @@
 class Business < ApplicationRecord
+	# Todo update method name to reflect search criteria
+	# Todo Add weights to columns
+	# Additonal options => normalization, sort_only, any_word
+	#* Default results are ordered by relevance
+	include PgSearch::Model
+	pg_search_scope :search_by_theme,
```
Comment (bpleslie) [inline app/models/business.rb]: Are you still using the scope strategy?

File: app/controllers/api/v1/businesses_controller.rb (hunk 1 above)
Comment (juliankohlman) [reply app/controllers/api/v1/businesses_controller.rb]: Gotcha Brad, I had to turn off prettier's `use tabs` option.

File: app/models/business.rb (hunk 2 above)
Comment (juliankohlman) [reply app/models/business.rb]: I ended up sticking with it for now. I made a branch for the multi-search, and when comparing the two the scope strategy seemed to get the job done. I want to circle back though and integrate [tsvector columns](https://github.com/Casecommons/pg_search#using-tsvector-columns) for some speed 🐎

**Pull request merged**